src/plugins/legacy/medVtkView/medVtkView.cpp

```diff
@@ -257,7 +257,9 @@ QPointF medVtkView::mapWorldToDisplayCoordinates(const QVector3D & worldVec)
     // The following code is implemented without calling ren->SetWorldPoint,
     // because that generates an unnecessary modified event.
 
-    vtkRenderer * ren = d->currentView->GetRenderer();
+    vtkRenderer * ren = d->view2d->GetRenderer();
+    if ( !ren )
```

Comment:
can you add brackets to these two blocks?

Reply:
no problem :-)
I did not add thzm to stay consistent with the other if/return in the functions do you want me to add brackets around them too?